packages/round-manager/src/features/api/types.ts

```diff
@@ -150,6 +150,14 @@ export interface Round {
    * Addresses of wallets that will have admin privileges to operate the Grant program
    */
   operatorWallets?: Array<string>;
+
+  /**
+   * Content of Grant Explorer application page
+   */
+  eligibility?: {
+    description: string;
+    requirements: { requirement: string }[];
```

Comment:
note: `requirements` needed to be an array of objects to use the `useFieldArray` helper - `useFieldArray` apparently doesn't work with arrays of primitives like `string[]`